static/src/stylesheets/module/_release-message.scss

```diff
@@ -504,6 +504,64 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 /*
  * Prominent membership message as a feature
  */
+
+
+
+.membership_paypal-logo {
+    display: none;
+    height: auto;
+    margin-right: $gs-gutter;
+    object-fit: contain;
+    order: 2;
+    margin-top: $gs-gutter/2;
+    margin-bottom: 0;
+    width: 200px;
+
+    @include mq($from: tablet) {
```

Comment:
`$from` media queries are the default case, so you can just do `@include mq(tablet) {` here and elsewhere